IntegrationWithPharmacies/TenderApi/Service/SmptServerService.cs

```diff
@@ -61,7 +61,7 @@ private static void SendMail(String filePath, String type)
             SmtpClient SmptServer = FormSmptServerInformation();
```

Comment:
Change name of SmtpClient object "SmptServer" into "smptServer" so it starts with a lower letter.